### PR TITLE
Remove confusing punctuation

### PR DESCRIPTION
**Pull request by brylie**
The text inside of the quote represents a WordPress version number. However, the trailing period is, while grammatically correct, confusing when reading the error output.

## Description

Remove confusing punctuation inside of quote.

## Screenshots 

image

## Types of changes

Small punctuation fix.

## Checklist:
- [ ] My code is tested.
- [ ] My code follows the WordPress code style. 
- [ ] My code follows the accessibility standards. 
- [ ] I've tested my changes with keyboard and screen readers. 
- [ ] My code has proper inline documentation. 
- [ ] I've included developer documentation if appropriate. 
- [ ] I've updated all React Native files affected by any refactorings/renamings in this PR (please manually search all `*.native.js` files for terms that need renaming or removal).

### CODE REVIEW

File: packages/env/lib/config/parse-config.js [hunk 1]
```diff
@@ -133,7 +133,7 @@ function parseSourceString( sourceString, { workDirectoryPath } ) {
 	}
 
 	throw new ValidationError(
-		`Invalid or unrecognized source: "${ sourceString }."`
+		`Invalid or unrecognized source: "${ sourceString }"`
```
Comment (ZebulanStanphill) [inline packages/env/lib/config/parse-config.js]: Wouldn't it make the most sense to just move the period to the end of the sentence?
```suggestion
		`Invalid or unrecognized source: "${ sourceString }".`
```

**Review by ZebulanStanphill — APPROVED** [review]

**Comment by Mamaduka** [comment]
Hi, @brylie

Can you rebase on top of the current trunk? E2E tests issue should be fixed now.

**Comment by brylie** [comment]
This is an example of why it would be beneficial to separate the WP-env project from the Gutenberg repository. A cross-repo rebase operation is difficult for new contributors and seems disproportionate for a one-line change to documentation.

**Comment by Mamaduka** [comment]
@brylie, I think that makes sense.

I don't have the information if this was discussed before, but @gziolo might have more context.

**Pull request merged**